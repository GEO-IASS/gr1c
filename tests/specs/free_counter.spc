ENV:;
SYS: y [0,4];

SYSINIT: y = 0;
SYSTRANS:   [] (y=0) -> (y'=2 | y'=4)
          & [] (y < 4);
SYSGOAL: []<> (y=3);
