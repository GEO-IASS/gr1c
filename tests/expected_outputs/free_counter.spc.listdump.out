   0 : 03 -  0 -  0 - [ 0]
   1 : 02 -  0 -  1 - [ 0]
   2 : 00 -  0 -  2 - [ 1]
   3 : 10 -  0 -  2 - [ 1]
